### PR TITLE
add/change regen rates for many units

### PR DESCRIPTION
**Pull request by andy5995**
Gave some air units "3" (similar to air units in MegaPack), some land
vehicles gave "2", also added some land vehicles to the technician's
repair capabilities

### CODE REVIEW

File: factions/enemies_and_allies/units/technician/technician.xml [hunk 1]
```diff
@@ -245,30 +245,35 @@
...
-				<unit name="oil_well"/>	
-				<unit name="military_research_centre"/>		
-				<unit name="headquarters"/>	
+				<unit name="barracks_i"/>
+				<unit name="barracks_i_i"/>
+				<unit name="barracks_i_i_i"/>
+				<unit name="hanger_i"/>
+				<unit name="hanger_i_i"/>
+				<unit name="hanger_i_i_i"/>
+				<unit name="leclerc_tank"/>
```
Comment (andy5995) [inline factions/enemies_and_allies/units/technician/technician.xml:261]: I added some new vehicles here...

File: factions/enemies_and_allies/units/hospital/hospital.xml [hunk 2]
```diff
@@ -57,26 +57,6 @@
 			<sound enabled="false" />
 		</skill>
 
-		<skill>
```
Comment (andy5995) [inline factions/enemies_and_allies/units/hospital/hospital.xml:60]: You can see the section I removed here weren't being used and I didn't anticipate that they would be.

**Comment by Jammyjamjamman** [comment]
advanced barracks are a starter unit, can you revert that?

**Comment by andy5995** [comment]
> advanced barracks are a starter unit, can you revert that?

What do you mean? I didn't make any changes to the advanced barracks... ??

**Comment by Jammyjamjamman** [comment]
my mistake, all good :-)

**Pull request merged**